test/unit/spec/v1/EnrollCall_spec.js

```diff
@@ -545,6 +547,7 @@ Expect.describe('EnrollCall', function() {
               },
             },
           ]);
+          expect(dispatchEventSpy).not.toHaveBeenCalled();
```

Comment:
Do we need a test where we check that the event is dispatched?

Reply:
don't think so? the expectation is that it's not called, i.e., that it doesn't try to report to Sentry

Reply:
I meant to say with an invalid code. As in, test that the event is dispatched when the conditions for that happen

Reply:
I meant to say with an invalid code. As in, test that the event is dispatched when the conditions for that happen

Reply:
I think such test should be added to `courage`